rs/sns/governance/src/proposal/advance_sns_target_version.rs

```diff
@@ -176,7 +176,7 @@ fn test_validate_and_render_advance_target_version_action() {
 
 ### Monitoring the upgrade process
 
-Please note: the upgrade steps above (valid around timestamp 0 seconds) might change during this proposal's voting period. Such changes are unlikely and are subject to NNS community's approval.
+Please note: the upgrade steps above (valid around 1970-01-01 00:00:00 UTC) might change during this proposal's voting period.
```

Comment:
[nitpick] same as before
```suggestion
Please note: the upgrade steps mentioned above (valid around 1970-01-01 00:00:00 UTC) might change during this proposal's voting period.
```